src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -2,27 +2,14 @@
 
 import bio.terra.model.SnapshotBuilderConcept;
 import bio.terra.model.SnapshotBuilderGetConceptsResponse;
-import bio.terra.model.SnapshotBuilderSettings;
 import java.util.List;
 import java.util.UUID;
 import org.springframework.stereotype.Component;
 
 @Component
 public class SnapshotBuilderService {
-  private final SnapshotBuilderSettingsDao snapshotBuilderSettingsDao;
 
-  public SnapshotBuilderService(SnapshotBuilderSettingsDao snapshotBuilderSettingsDao) {
-    this.snapshotBuilderSettingsDao = snapshotBuilderSettingsDao;
-  }
-
-  public SnapshotBuilderSettings getSnapshotBuilderSettings(UUID datasetId) {
-    return snapshotBuilderSettingsDao.getSnapshotBuilderSettingsByDatasetId(datasetId);
-  }
-
-  public SnapshotBuilderSettings updateSnapshotBuilderSettings(
-      UUID id, SnapshotBuilderSettings settings) {
-    return snapshotBuilderSettingsDao.upsertSnapshotBuilderSettingsByDataset(id, settings);
-  }
+  public SnapshotBuilderService() {}
```

Comment:
This currently has nothing, but will have more dependencies as we add more actual snapshot builder service things.

Reply:
Not a deal breaker, but in that case I'd remove it for now and rely on the default constructor.